### PR TITLE
address length problem (issue #2649)  fixed

### PR DESCRIPTION
**Pull request by magdalenesuo**
Issue #2649 fixed- added tooltip

(https://cloud.githubusercontent.com/assets/20799954/21082256/ddc17b62-bffd-11e6-9e70-ca8efdc9c7a1.png)

### CODE REVIEW

File: app/templates/gentelella/index.html [hunk 1]
```diff
@@ -41,12 +43,18 @@
...
+            transition: background-color 0.5s;
+            height: 18px; 
+            padding-right: 10px;
         }
 
         .event-holder .caption {
             background-color: #FFFFFF;
             transition: background-color 0.5s;
+            height:140px;
+
```
Comment (SaptakS) [inline app/templates/gentelella/index.html]: Please remove the unwanted spaces.

**Comment by niranjan94** [comment]
@magdalenesuo multi-line location doesn't look good. The event box looks tall and many times the location name is only a single line. 

Instead of increasing the height to accommodate multiple lines, keep the height as it is, and if the location name goes beyond a length, maybe ellipsize  it and show it as a tooltip or something ? 

Most of the use cases, the user would be filtering for events close by to his location and anyway only first few words of the location name is gonna matter to the user 

@SaptakS what do you feel ?

**Comment by niranjan94** [comment]
> Most of the use cases, the user would be filtering for events close by to his location and anyway only first few words of the location name is gonna matter to the user 

For example,

If the location name is _Science Centre, 15 Science Centre Rd, Singapore_ , 
for a user who is searching for events in singapore, only __Science Centre_ is gonna matter.

File: app/templates/gentelella/index.html (hunk 1 above)
Comment (magdalenesuo) [inline app/templates/gentelella/index.html]: removes

File: app/templates/gentelella/index.html [hunk 2]
```diff
@@ -142,7 +150,9 @@
                     <small class="time">{{ event.start_time.strftime('%a, %B %d %I:%M %p') }}</small>
                     <h4 class="name">{{ event.name }}</h4>
                     <small class="location_name">{{ event.location_name.split(",")[0] }}</small>
-                    <hr>
+                    
+                </div>
+                <hr>
                     <p class="tags">
```
Comment (magdalenesuo) [inline app/templates/gentelella/index.html]: yes, fixed

**Comment by SaptakS** [comment]
@niranjan I agree with you but the problem is sometimes the first few part of the location itself goes beyond length. In that case problem arises. If that doesn't happen there is no problem anyways. So not really sure what should be the thing. Tooltip doesn't see good. We may just truncate in that case?

**Comment by SaptakS** [comment]
> If the location name is Science Centre, 15 Science Centre Rd, Singapore,
for a user who is searching for events in singapore, only Science Centre is gonna matter.

Exactly. In fact if it is comma separated it doesn't create an issue at all. The issue is only if someone write like Science Centre 15 Science Centre Rd Singapore or if the name itself is that big. Then we have a problem.

I think we should be giving a fixed size and not allow the box to have different sizes. The only question is whether to make it shorter or longer?

**Comment by SaptakS** [comment]
@niranjan what do you think?

**Comment by niranjan94** [comment]
@SaptakS I'm not talking about a split by comma ... Just ellipsize it when it goes beyond the width of the box (I have used a js function to do that in the scheduler). Then add a title attribute to the span which has the location name with the full location. So, when a user hovers on the ellipized location name, he/she will be able to see the full name as a native browser tooltip

**Comment by SaptakS** [comment]
> So, when a user hovers on the ellipized location name, he/she will be able to see the full name as a native browser tooltip

That's a good idea. Only I am not very sure whether a tooltip is necessary.

**Comment by magdalenesuo** [comment]
@niranjan94  Oh I didn't know about it. Thanks , will look into it :)

File: app/templates/gentelella/index.html [hunk 3]
```diff
@@ -141,7 +163,8 @@
                 <div class="caption">
                     <small class="time">{{ event.start_time.strftime('%a, %B %d %I:%M %p') }}</small>
                     <h4 class="name">{{ event.name }}</h4>
-                    <small class="location_name">{{ event.location_name.split(",")[0] }}</small>
+                    <div class="location_name">{{ event.location_name.split(",")[0] }}</div>
+                    <span class="locate">{{ event.location_name.split(",")[0] }}</span>
```
Comment (SaptakS) [inline app/templates/gentelella/index.html]: Inside locate, show the entire event.location.

File: app/templates/gentelella/index.html [hunk 4]
```diff
@@ -24,7 +24,7 @@
         }
 
         .event-holder .thumbnail {
-            padding: 0;
+            padding: 0px;
```
Comment (SaptakS) [inline app/templates/gentelella/index.html]: Please change 0px to 0... 0 has no unit...

**Comment by magdalenesuo** [comment]
@SaptakS @niranjan94 I have made the changes

File: app/templates/gentelella/index.html (hunk 3 above)
Comment (magdalenesuo) [inline app/templates/gentelella/index.html]: changes made

Comment (magdalenesuo) [inline app/templates/gentelella/index.html]: (https://cloud.githubusercontent.com/assets/20799954/21082256/ddc17b62-bffd-11e6-9e70-ca8efdc9c7a1.png)

**Pull request merged**